pages/Ethereum/Overview.md

```diff
@@ -15,10 +15,11 @@ Other versions may be available in the [official contracts code repository](http
 Official DSNP interfaces, contracts, and deployment information may be found in [GitHub](https://github.com/LibertyDSNP/contracts).
 
 ## Libraries
-
+<!-- yaspeller ignore:start -->
```

Comment:
and if yaspeller is being unreasonable this is how you tell it to skip something.